VocabularyTrainer/Home Screen/AboutViewController.swift

```diff
@@ -0,0 +1,194 @@
+//
+//  AboutViewController.swift
+//  VocabularyTrainer
+//
+//  Created by skrr on 04.01.23.
+//  Copyright © 2023 mic. All rights reserved.
+//
+
+import UIKit
+import WebKit
+
+/// Shows privacy policy loaded from an HTML located in the Github repo.
+final class AboutViewController: UIViewController {
+    /// The webview showing the html.
+    private let webView = WKWebView(frame: .zero)
+    /// The loading alert.
+    private var loadingController: UIAlertController?
+    /// The string added to the html's url to get the correct PP for the user's locale.
+    private var langCodeString: String {
+        var langString = "en"
+        if let languageCode = Locale.current.languageCode {
+            if languageCode == "pt" {
+                langString = "pt-BR"
+            } else {
+                langString = languageCode
+            }
+        }
+        return langString
+    }
+    /// The PP's location.
+    private var urlString: String {
+        "https://htmlpreview.github.io/?https://github.com/misteu/VocabularyTraining/blob/master/VocabularyTrainer/\(langCodeString).lproj/pp.html"
+    }
+
+    /// Close button for the navbar, shown when initial url (`urlString`) is loaded.
+    private lazy var closeButton: UIBarButtonItem = {
+        let button = UIBarButtonItem(title: Strings.backButtonTitle,
+                                     image: UIImage(systemName: "x.circle"),
+                                     primaryAction: .init(handler: { [weak self] _ in
+            self?.dismiss(animated: true)
+        }), menu: nil)
+        button.tintColor = HomeViewModel.Colors.flippyGreen
+        return button
+    }()
+
+    /// Back button for the navbar, shown when navigated away from `urlString`.
+    private lazy var backButton: UIBarButtonItem = {
+        let button = UIBarButtonItem(title: Strings.backButtonTitle,
+                                     image: UIImage(systemName: "chevron.left"),
+                                     primaryAction: .init(handler: { [weak self] _ in
+            self?.webView.goBack()
+            }), menu: nil)
+        button.tintColor = HomeViewModel.Colors.flippyGreen
+        return button
+    }()
+
+    /// Label showing the version number of the app.
+    private let versionLabel: UILabel = {
```

Comment:
Suggestion: use the UILabel extension.

Reply:
Updated that (see a19a2958c771f844d9f71ab443d46203e76011fd): https://github.com/misteu/VocabularyTraining/blob/a19a2958c771f844d9f71ab443d46203e76011fd/VocabularyTrainer/Home%20Screen/AboutViewController.swift#L69

I also added some documentation to the extension.